src/applications/mhv-landing-page/mocks/api/user/index.js

```diff
@@ -135,6 +137,7 @@ const generateUserWithFacilities = ({ facilities = [], name = 'Harry' }) => {
           ...defaultUser.data.attributes.profile,
           facilities,
           first_name: name,
+          mhvAccountState: 'OK',
```

Comment:
I believe this should be under `va_profile` and not `profile`

Reply:
Also, seems like having `facilities` here is a mistake. That should also be under `va_profile`, AFAICAT.